### DISCUSSION TITLE
是否可以支持一次分析四名选手？

### DISCUSSION THREAD

**Discussion by Preserver6166** [Equim-chan/mjai-reviewer#131] [Ideas]
在目前的流程下需要操作四次，而且会展示为4个不同的分析结果页面。理想情况是在分析结果页面上有一个sidebar可以切换不同选手的视角。
另外问一下目前的牌谱分析结果有无retention time？或者这方面的计划？

**Comment by Equim-chan** [comment]
不计划支持同时分析多个视角的功能，因为我期望 mjai-reviewer 所需输入的信息越少越好，例如目前在没有完整牌山和对手手牌信息的情况下仍然能进行分析（常见于根据何切截图倒推），而同时分析多个视角则需要更完整的数据。

目前在线版的牌谱分析结果会保留 15 天，这一点在网站下方的 FAQ 里有提到。

**Reply by Preserver6166** [reply]
好的，了解了[抱拳][抱拳][抱拳]

**Comment by chucklu** [comment]
感觉这是一个比较实用的功能，有时候吃了四位，想要看一下其他对手的牌谱信息。需要重新弄一遍提交流程，有点麻烦。
